### PR TITLE
Refactor: `getTitle` utility function

### PR DESCRIPTION
**Pull request by kaitmore**
- Refactored `getTitle` to grab title data from pathname
- Deleted instances of `PageTitle` (now obsolete)

Accessing specific array indices to get the "service : instance" title still seems fragile to me, but I'm not sure of a better solution atm.

### CODE REVIEW

File: src/components/AppHeader/index.js [hunk 1]
```diff
@@ -2,56 +2,45 @@ import { PropTypes } from "prop-types";
...
 
-function mapStateToProps(state) {
-  return {
-    selectedService: state.settings.selectedService,
-    selectedInstance: state.settings.selectedInstance
-  };
+  // If there is an instance in the path, display root : instance
+  // If not, then check if the root is truthy and display that
+  // Else just display Fabric
+  return instance ? `${root} : ${instance}` : root ? root : "Fabric";
```
Comment (spmcbride1201) [inline src/components/AppHeader/index.js]: This is a pretty scary line of code. Nested ternaries give me nightmares. Can you use if statements or something cleaner here?

File: src/components/AppHeader/index.js [hunk 2]
```diff
@@ -2,56 +2,45 @@ import { PropTypes } from "prop-types";
...
     </AppHeaderContainer>
   );
 }
 
-function getTitle(pathname, selectedService, selectedInstance) {
-  // This will display the first item in the path
-  // Else if we are on an instance, it will display serviceName : last 8 digits of instanceID
-  // If the path is on the root, display Fabric
+function getTitle(pathname) {
   const path = pathname
```
Comment (spmcbride1201) [inline src/components/AppHeader/index.js]: This seems like a good place for array destructuring assignments and default values. You should be able to merge 37-38 into the line in place of path.

File: src/components/AppHeader/index.js [hunk 3]
```diff
@@ -2,56 +2,45 @@ import { PropTypes } from "prop-types";
 import React from "react";
 import { getFabricServer } from "../../utils/head";
```
Comment (spmcbride1201) [inline src/components/AppHeader/index.js]: Please refactor these crazy imports using the new syntax for all of the imports in the files you touch.

**Comment by kaitmore** [comment]
@spmcbride1201 Addressed your comments

**Pull request merged**